### PR TITLE
DTJ Chapter 3

### PR DESCRIPTION
**Pull request by thatguy59**
Just the basics for now, I might do more

### CODE REVIEW

File: src/dtj/java/com/lambdaherding/edi/dtj/ch03/Exercise01.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	 * =================================================================
+	 * Question 3
+	 * =================================================================
+	 */
+
+	public static void q3() {
+		System.out.println( "Question 3" );
+		List<Album> albums = threeOrLess( Stream.of(
+				new Album()
+					.setTitle( "Marks Songs to Twerk to" )
```
Comment (a-panayotopoulos) [inline src/dtj/java/com/lambdaherding/edi/dtj/ch03/Exercise01.java:68]: :frowning_face:

**Review by a-panayotopoulos — APPROVED** [review]
:dog:

File: src/dtj/java/com/lambdaherding/edi/dtj/ch03/Exercise01.java (hunk 1 above)
Comment (utopia-2) [reply src/dtj/java/com/lambdaherding/edi/dtj/ch03/Exercise01.java:68]: Ah, you too remember that fateful evening.

File: src/dtj/java/com/lambdaherding/edi/dtj/ch03/Exercise01.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+					.setTracksCount(7),
+				new Album()
+					.setTitle( "Reds Crazy Synths" )
+					.setTracksCount(3),
```
Comment (utopia-2) [inline src/dtj/java/com/lambdaherding/edi/dtj/ch03/Exercise01.java:72]: 3 glorious 30 minute tracks

**Pull request merged**